### ISSUE TITLE
[Accessibility] Avoid repetition of identical link text on same page

### ISSUE THREAD

**Issue by hostle83** [DSpace/dspace-angular#3083]
**Describe the bug**
This relates to items that have a number of different download options. For each option, the Download Link currently says "Download". According to Success Criterion 2.4.4 Link Purpose (In Context), the user should be able to determine the purpose of each link from the link text alone. [In this example](https://sandbox.dspace.org/items/cd374806-70ac-4541-a711-559c9bb60a76/full), users with screen readers would hear the word "Download" read numerous times but not know the context, purpose and destination of each download link.

**To Reproduce**
1. Visit https://sandbox.dspace.org/items/cd374806-70ac-4541-a711-559c9bb60a76/full
2. See list of Download Links at the right side of each file description

**Expected behavior**
A clear and concise description of what each link does, e.g. "Download the PDF...". 

**Related work**

**Comment by Andrea-Guevara** [comment]
Good morning @tdonohue, my name is Andrea. I'm part of the Neki-it company. We'd like to take on this activity.

**Comment by Andrea-Guevara** [comment]
Good morning @tdonohue! I hope you're well

We've provided a PR with a possible solution to this problem https://github.com/DSpace/dspace-angular/pull/3264. We look forward to your feedback.
